bee-common/bee-packable/src/packable/vec_prefix.rs

```diff
@@ -18,6 +18,8 @@ use core::{convert::TryFrom, marker::PhantomData};
 /// the length of the source vector exceeds the maximum length of the [`VecPrefix`].
 #[derive(Debug, PartialEq, Eq)]
 pub struct PrefixedFromVecError {
```

Comment:
Can't this be just `InvalidBoundedU32` and so on?

Reply:
Good point -- will change that 👍